charts/pvc-autoresizer/templates/controller/deployment.yaml

```
@@ -33,6 +33,7 @@ spec:
           args:
             - --prometheus-url={{ .Values.controller.args.prometheusURL }}
             - --interval={{ .Values.controller.args.interval }}
+            - --use-k8s-metrics-api={{ .Values.controller.args.useK8sMetricsApi }}
```

```suggestion
          {{- if .Values.controller.args.useK8sMetricsApi }}
            - --use-k8s-metrics-api={{ .Values.controller.args.useK8sMetricsApi }}
          {{- end }}
```

We will be able to avoid [the CI failure](https://github.com/topolvm/pvc-autoresizer/actions/runs/7812272392/job/21308980919?pr=242) by this change.